lib/src/models/option_item.dart

```diff
@@ -8,13 +8,13 @@ class OptionItem {
     this.subtitle,
   });
 
-  Function()? onTap;
+  Function(BuildContext context)? onTap;
```

Comment:
Two things:

1. Make all of this class's properties `final`.
2. Rename this function to:

```dart
final void Function(BuildContext context) onTap;
```